### ISSUE TITLE
Kategorie: Kinderboulder

### ISSUE THREAD

**Issue by PeterGrabowitz** [LocalCrag/LocalCragApp#69]

**Comment by dorthrithil** [comment]
Wäre das ein Boolean einer Linie? Bewertung ist dann trotzdem nach normaler Skala, ne? Gibt's auch sowas wie "kinderrouten"? LC soll ja auch potentiell für Seilkletterei funktionieren. Vllt nenn man das dann "Kindertauglich" oder so

**Comment by PeterGrabowitz** [comment]
Ja, genau, "kindertauglich" ist im Grunde die allgemeingültige Kategorie. Letztlich geht es vor allem um Griffabstände, weniger die schiere Schwierigkeit, aber klar, die ist meistens dann nicht hoch.
